CRM/Odoosync/Upgrader.php

```diff
@@ -119,4 +129,86 @@ private function getDefaultOptionValue($optionGroupName) {
     return (int) $value['values'][0]['value'];
   }
 
+  /**
+   * Install default message template
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function installMessageTemplate() {
+
+    $filePath = $this->extensionDir . "/templates/CRM/Odoosync/DefaultMessageTemplates/OdooSyncErrorReport.html";
+
+    $messageHtml = '';
+    if (file_exists($filePath)) {
+      $messageHtml = file_get_contents($filePath);
+    }
+    else {
+      CRM_Core_Session::setStatus(ts('Creating message template'), ts("Couldn't find default template at '$filePath'"), 'alert');
+    }
+
+    $workflowId = $this->getWorkflowId();
+
+    civicrm_api3('MessageTemplate', 'create', [
+      'msg_title' => "CiviCRM Odoo Sync Error Report",
+      'msg_subject' => ts("CiviCRM Odoo Sync Error Report"),
+      'is_reserved' => 0,
+      'msg_html' => $messageHtml,
+      'is_active' => 1,
+      'msg_text' => 'N/A',
+      'workflow_id' => $workflowId
+    ]);
+  }
+
+  /**
+   * Gets workflow id related to the message template
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function getWorkflowId() {
+    $id = civicrm_api3('OptionValue', 'getvalue', [
+      'return' => "id",
+      'option_group' => 'msg_tpl_workflow_odoo_sync',
+      'name' => 'civi_crm_odoo_sync_error_report',
+    ]);
+
+    return (int) $id;
+  }
+
+  /**
+   * Deletes the message template created by the extension
```

Comment:
please renae the description to : 

Deletes 'CiviCRM-Odoo Sync Error Report' message template